templates/vue/src/store/mutations.js

```diff
@@ -83,6 +83,7 @@ export function setDataset(state, dataset) {
       Object.values(value).forEach(node => {
         // Has to call this so `state.nodes` is reactive
         Vue.set(state.nodes, node.id, node)
+        node.childOrdering = [...new Set(node.childOrdering)]
```

Comment:
Why is this needed? 🤔

Reply:
I think my intention was that if there was already a duplicate (ie. childOrdering contained the same id twice), then it stayed a duplicate even after reloading so this line was supposed to fix that for already affected nodes, I can test again maybe it's not needed

Reply:
Update: I retested, turns out this line is unnecessary, that was an oversight on my part sorry about that

Reply:
No worries thanks for taking a look!